python/cudf/cudf/_lib/column.pyx

```diff
@@ -202,11 +202,13 @@ cdef class Column:
 
     def _clear_cache(self):
         self._distinct_count = {}
-        try:
-            del self.memory_usage
-        except AttributeError:
-            # `self.memory_usage` was never called before, So ignore.
-            pass
+        attrs = ("memory_usage", "is_monotonic_increasing", "is_monotonic_decreasing")
```

Comment:
Are there any places where we need to insert a `_clear_cache` call that didn't have one before now that these properties are also cached? I did a quick search and didn't find any, but wanted to confirm that you checked too.

Reply:
I just looked and it doesn't looks like there's anywhere we truly modify a Column inplace where these attributes might change (I suspected `__setitem__` but it appears it's "inplace" where the base data is swapped)